src/Sylius/Bundle/ImportExportBundle/Form/Type/Writer/ImportWriterChoiceType.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ImportExportBundle\Form\Type\Writer;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ * @author Bartosz Siejka <bartosz.siejka@lakion.com>
+ */
+class ImportWriterChoiceType extends AbstractType
```

Comment:
Same as `ImportReaderChoiceType` and `ExportReaderChoiceType` comes with `ImportWriterChoiceType` and `ExportWriterChoiceType`. They can be one class defined four times in DIC.